CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
+#### [v2.3.3]
```

Comment:
This is not the correct version that this change would be in. You can remove this section. I'll add it to the release notes of the version that this ends up making it into.